recipe/meta.yaml

```
@@ -37,15 +37,15 @@ requirements:
     - pip
     - setuptools
     - wheel
-    - llvmlite 0.43.0
-    - numpy 2.0.2
+    - llvmlite 0.44.0
+    - numpy 2.1.3
     - tbb-devel 2021.8.0
     - _openmp_mutex 5.1        # [linux]
   run:
     - python
-    - llvmlite >=0.43.0,<0.44.0a0
+    - llvmlite >=0.44.0,<0.45.0a0
     # avoid 2.0.1 because of https://github.com/numpy/numpy/pull/27195
-    - numpy >=1.22.3,<2.1,!=2.0.1
+    - numpy >=1.22.3,<2.2,!=2.0.1
```

```suggestion
    - numpy >=1.24,<2.2,!=2.0.1
```
https://github.com/numba/numba/blob/0.61.0/setup.py#L25